spec/quality_spec_es.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
```

Comment:
one last minor quibble: this file should be named `quality_es_spec.rb`, since we usually glob for `*_spec.rb`